pkg/gccbuilder/gcc-builder.go

```diff
@@ -69,6 +69,7 @@ var (
 		"-undef":           struct{}{},
 		"-nostartfiles":    struct{}{},
 		"-remap":           struct{}{},
+		"-r":               struct{}{},
```

Comment:
Have you checked, if this actually produces the correct results? I didn't knew about this incremental linking feature. I am worried about loosing the data that is already stored in the db.

Reply:
Yes we are loosing information if we remove the -r flag. The -r generates a relocatable output file that can in turn serve as input to ld..

Reply:
In the end, if we clean the -r flag we still keep the information for the relocatable output file as it's followed by linking information of the relocatable file linked to the libraries e.g. -r -o librtld.map.o dl-allobjs.os libc_pic.a